modules/apps/31-ibc-query/keeper/events.go

```diff
@@ -1 +1,26 @@
 package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/cosmos/ibc-go/v4/modules/apps/31-ibc-query/types"
+)
+
+// EmitCreateClientEvent emits a create client event
+func EmitQueryEvent(ctx sdk.Context, query *types.MsgSubmitCrossChainQuery) {
+	ctx.EventManager().EmitEvents(sdk.Events{
```

Comment:
It is not critical but I heard that EmitEvents is deprecated.
How about using EmitTypedEvent? We need to implement event type in proto.